packages/flutter/lib/src/material/slider.dart

```diff
@@ -197,6 +197,11 @@ class Slider extends StatefulWidget {
     this.focusNode,
     this.autofocus = false,
     this.allowedInteraction,
+    @Deprecated(
+      'Use SliderTheme to customize the Slider appearance. '
+      'This feature was deprecated after v3.26.0-0.1.pre.'
+    )
+    this.year2023 = false,
```

Comment:
Should we also make `year2023` default to true here?

Reply:
Yes, we'll match this with progress indicators.